script.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable */
+const remove = document.getElementById('remove');
+/* eslint-enable  */
+const add = document.querySelector('#add');
+
+let booksArray = [];
+
+/* eslint-disable */ 
+class bookApp {
+  static addBook() {
+    const section = document.getElementById('bookSection');
+    section.innerHTML = '';
+    for (let i = 0; i <= booksArray.length - 1; i++) { section.innerHTML += `<div id="textHeading"><div><p id="title">"${booksArray[i].title}" By </p> <p id="author">${booksArray[i].author}</p></div> <button id="remove" onclick="bookApp.removeBook(${i})">Remove</button></div>`; }
+  }
+  static removeBook(id) {
+    booksArray.splice(id, 1);
+    bookApp.addBook();
+    localStorage.setItem('booksArray', JSON.stringify(booksArray));
+  }
+}
+/* eslint-enable  */
+
+// Local Storage
+window.onload = () => {
+  if (localStorage.getItem('booksArray')) {
+    booksArray = JSON.parse(localStorage.getItem('booksArray'));
+  }
+  bookApp.addBook();
+};
+
+if (add) {
+  add.addEventListener('click', () => {
+    const title = document.getElementById('inputTitle');
+    const author = document.getElementById('inputAuthor');
+    const newBook = {
+      title: title.value,
+      author: author.value,
+    };
+    booksArray.push(newBook);
+    bookApp.addBook();
+    localStorage.setItem('booksArray', JSON.stringify(booksArray));
+    title.value = '';
+    author.value = '';
+  });
+}
```

Comment:
- [ ] Kindly move the functionality into a function for adding books. Your class is expected to have addBook, removeBook and displayBook functions. Currently, your main functionality for adding a book is outside the class. Kindly create a function for adding a book and move this functionality into it.

NOTE:
In your previous task, I suggested you rename your current addBook function to displayBook because that's what that function does currently. So now you can rename it to what I suggested and then the real addBook function can have this functionality for adding a book to the bookLibrary. 👍

Reply:
Thanks for the review. But this is only the event listener here. The main addbook functionality is in the class. I think so. Please guide me.

Reply:
Fixed.